docs/cards.md

```diff
@@ -138,19 +138,34 @@ Try hovering over then clicking on the cards below:
 
 :::{card} Clickable Card (external)
 :link: https://example.com
-:link-alt: example
+:link-alt: example.com
 
 The entire card can be clicked to navigate to <https://example.com>.
+
+For **external** link cards, if you do not provide `link-alt`, the URL will be
+used as the text of the link. Using URLs as link text makes it harder for
+disabled people and for search engines to digest your web page, so it's best to
+provide link text via the `link-alt` option.
 :::
 
 :::{card} Clickable Card (internal)
 :link: cards-clickable
 :link-type: ref
-:link-alt: cards-clickable
 
-The entire card can be clicked to navigate to the `cards-clickable` reference target.
+The entire card can be clicked to navigate to the `cards-clickable` reference
+target.
+
+For **internal** link cards, it's unnecessary to provide `link-alt`. The name of
+the section that you are referencing will be used as the link's accessible name.
+For example, this card references `cards-clickable`, so the link title will be
+"Clickable cards" (same as the section heading). If you wish to use a different
+title for the link, you can provide `link-alt` and it will take precedence.
 :::
 
+Note: you cannot add additional links to the clickable card, neither in the card
```

Comment:
I added this because I tried adding a link inside the card body of the external clickable card to a page explaining [why link text is important](https://business.scope.org.uk/article/how-to-write-better-link-text-for-accessibility) but clicking on the link took me to example.com 😆